posthog/models/cohort/util.py

```diff
@@ -277,6 +281,20 @@ def recalculate_cohortpeople(cohort: Cohort, pending_version: int) -> Optional[i
     return count
 
 
+def clear_stale_cohortpeople(cohort: Cohort, current_version: int) -> None:
+    stale_count_result = sync_execute(
+        STALE_COHORTPEOPLE,
+        {"cohort_id": cohort.pk, "team_id": cohort.team_id, "version": current_version},
+    )
+
+    if stale_count_result and len(stale_count_result) and len(stale_count_result[0]):
+        stale_count = stale_count_result[0][0]
+        if stale_count > 0:
+            AsyncDeletion.objects.create(
```

Comment:
This will create duplicates every 120 seconds right?

Reply:
changed to `get_or_create`